content/rs/security/encryption/tls/enable-tls.md

```diff
@@ -65,11 +69,14 @@ To enable TLS for client connections:
 
 1. Select **Save**.
 
-1. _(Optional)_ By default, Redis Enterprise Software validates client certificate expiration dates.  You can use `rladmin` to turn off this behavior.
+    {{< note >}}
+By default, Redis Enterprise Software validates client certificate expiration dates.  You can use `rladmin` to turn off this behavior.
 
     ```sh
     rladmin tune db < db:id | name > mtls_allow_outdated_certs enabled
     ```
+    
+    {{< /note >}}
```

Comment:
This note block is not displaying correctly. Removing the leading whitespace for all lines of this note might fix it.